dev/make/compiler_definitions/gnu.mk

```
@@ -28,6 +28,9 @@ CORE.SERV.COMPILER.gnu = generic
 -Zl.gnu =
 -DEBC.gnu = -g
 
+-asanstatic.gnu = -static-libasan
```

```suggestion
-asanstatic.gnu =
```

It at least compiles successfully if this argument is removed. Not sure about the impact though.

---

Thanks for checking, it looks like by not including it, it will default to dynamic linking (https://gcc.gnu.org/onlinedocs/gcc/Link-Options.html#index-static-libasan).  Seems like an issue with GCC, should we leave a note and remove it, saying that static linking with GNU compilers is temporarily unavailable?